sdv/constraints/tabular.py

```diff
@@ -40,8 +40,8 @@
 from sdv.constraints.errors import (
     AggregateConstraintsError, ConstraintMetadataError, FunctionError, InvalidFunctionError)
 from sdv.constraints.utils import (
-    add_nans_column, cast_to_datetime64, logit, matches_datetime_format, revert_nans_columns,
-    sigmoid)
+    add_nans_column, cast_to_datetime64, get_datetime_diff, logit, matches_datetime_format,
```

Comment:
do we still need the logit and sigmoid functions?

Reply:
Yes, for the `ScalarRange` constraint. However, we could also use the logic of the `Inequality` constraint for the `ScalarRange` constraint I think.